cpp/tests/strings/json_tests.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/scalar/scalar_factories.hpp>
+#include <cudf/strings/substring.hpp>
+
+#include <cudf_test/base_fixture.hpp>
+#include <cudf_test/column_wrapper.hpp>
+
+struct JsonTests : public cudf::test::BaseFixture {
+};
+
+// json array
+  const char* str1 = "[\n\t\t{\n\t\t\t\"category\": \"reference\",\n\t\t\t\"author\": "
```

Comment:
Instead of `\r\t\n` like that, please use string literal as it would be much readable.